client/post-editor/editor-action-bar/style.scss

```diff
@@ -13,6 +13,18 @@
 	}
 }
 
+.editor-action-bar__cell {
+	width: 33%;
```

Comment:
I think rather than explicitly assigning width as thirds for each cell, if our intent is to grant equal width to each cell, we could leverage Flexbox, i.e. `flex: 1`.